src/frontend/ExtensionManager/index.tsx

```diff
@@ -42,7 +43,7 @@ const ExtensionManager = function () {
   }
 
   if (isOverlay) {
-    return <ExtensionPopup />
+    return <ExtensionContents />
```

Comment:
This was the problem, we were showing the normal MM popup regardless of the state in the ExtensionStore